templates/individual_response/post-confirmation.html

```diff
@@ -0,0 +1,7 @@
+{% extends 'individual_response/question.html' %}
```

Comment:
The template names confused me as there are two confirmations. I think this would be better named `confirm-address.html`.

Reply:
actually switched these around now due to the contact us text being in the wrong template:

there is now:
	- question.html
	- interstitial.html
	- confirmation.html
	- how.html

where how inherits from individual_response/question.html